.eslintrc.yaml

```diff
@@ -110,6 +110,9 @@ rules:
   }, {
     selector: "ThrowStatement > CallExpression[callee.name=/Error$/]",
     message: "Use new keyword when throwing an Error."
+    }, {
+    selector: "CallExpression[callee.object.name='assert'][callee.property.name='fail'][arguments.length=1]",
+    message: "assert.fail() message should be third argument"
```

Comment:
This is really clear if you did `assert.fail('message')`, but not if you made another mistake (i.e. did `assert.fail(false)` or something. Maybe something more generic like `assert.fail() should take at least three arguments`?